crates/sui-core/src/authority/authority_per_epoch_store.rs

```diff
@@ -274,7 +274,7 @@ pub struct AuthorityEpochTables {
     user_signatures_for_checkpoints: DBMap<TransactionDigest, Vec<GenericSignature>>,
 
     /// Maps sequence number to checkpoint summary, used by CheckpointBuilder to build checkpoint within epoch
-    builder_checkpoint_summary: DBMap<CheckpointSequenceNumber, CheckpointSummary>,
+    builder_checkpoint_summary: DBMap<CheckpointSequenceNumber, BuilderCheckpointSummary>,
```

Comment:
i guess we will just not push this to testnet, right? we could do this compatibly if we needed to but its not worth the effort.